opendatahub-io/odh-dashboard
# Add permissions for templates and serving runtimes

## Description


Closes #1464 

## How Has This Been Tested?




### Deploy the image
1. Switch to this branch
2. Change the `IMAGE_REPOSITORY` env variable in your `env.local` file to `IMAGE_REPOSITORY=quay.io/opendatahub/odh-dashboard:pr-1466`
3. Make sure you don't have the `dashboard` resource as a component in the operator `kfdef`
4. Run `make deploy`

### Test changes
1. Go to `Settings - Serving Runtimes`
2. Click “Add new”
3. Check that you can create a new Serving Runtime
4. Try to edit it
5. Try to delete it

## Test Impact


Current testing efforts are fine, this is testing dashboard service account permissions and we are excluding backend testing for now.

## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
